Kalkulator HTML/izgled.html

```diff
@@ -4,9 +4,17 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Digitron prvoklasni</title>
+    <style>
+        h1 {
+            text-align: center;
+        }
+    </style>
+
 </head>
 <body>
-    
+    <h1>Ovo je nikad jaci kalkulator</h1>
+    <center><img src="https://digitalcampaignsstorage.blob.core.windows.net/sugar-calculator/production/images/calculatorillustration.jpg" alt="Digitron" width="500" height="300">
```

Comment:
What is the center tag? I would use div and CSS flexbox for the grid, it will be helpful to center the image.

Flexbox: https://css-tricks.com/snippets/css/a-guide-to-flexbox/

Flexbox game: https://flexboxfroggy.com/